pulse/broker/src/main/java/gov/ca/emsa/pulse/service/AlternateCareFacilityService.java

```diff
@@ -78,19 +92,21 @@ public AlternateCareFacility create(@RequestBody(required=true) AlternateCareFac
 
 	@ApiOperation(value = "Edit an existing ACF")
 	@RequestMapping(value = "/{acfId}/edit", method = RequestMethod.POST)
+	@PreAuthorize("hasPermissionForAcf(#acfId)")
 	public AlternateCareFacility update(@PathVariable("acfId") Long acfId, 
 			@RequestBody(required=true) AlternateCareFacility toUpdate)
 		throws InvalidArgumentsException, PermissionDeniedException, 
 			AcfChangesNotAllowedException, SQLException {
 		
 		CommonUser user = UserUtil.getCurrentUser();
 		//auditManager.addAuditEntry(QueryType.EDIT_ACF, user.getAcf() + "/edit", user.getSubjectName());
-		if(user.getAcf() == null) {
```

Comment:
Can just delete this commented out section. No longer needed, and git is great about history